docker-builder.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-scriptdir="$(cd "${0%/*}"; pwd)"
-${scriptdir}/tools/docker/build.sh "$@"
+scriptdir="$(cd "${0%/*}" || exit 1; pwd)"
```

Comment:
Like for the other PR, I think `$(cd "${0%/*}" && pwd)` is less confusing (exit might make you think it exits the script)
(nitpick)